project/src/store/action.ts

```diff
@@ -0,0 +1,10 @@
+import {createAction} from '@reduxjs/toolkit';
+
+export const getOffersByCity = createAction('offers/cityChange', (city: string) => ({
+  payload: city,
+}));
+
+export const changeOffersSort = createAction('offers/changeOffersSort',
+  (sort: string) => ({
```

Comment:
экшены нужны что бы что то сделать со стором, поэтому в названиях обычно не используют get, только set, 
```suggestion
export const setActiveCity = createAction('offers/cityChange', (city: string) => ({
  payload: city,
}));

export const changeActiveSortType = createAction('offers/changeOffersSort',
  (sort: string) => ({
```